app/views/templates/report-details.html.slim

```diff
@@ -6,10 +6,10 @@
     .label.label-primary ng-if='!report' Loading...
     span ng-repeat='project in report.projects'
       span.label.label-primary ng-repeat='version in report.projectVersions | filter: {projectId: project.id}' {{ project | projectName }} {{ version.name }}
-      |  
+      |
```

Comment:
Replace these `|` by `'` to make sure the extra spaces remain (2 other occurrences below).